tests/baselines/reference/commentOnAmbientClass1.js

```diff
@@ -1,7 +1,12 @@
 //// [tests/cases/compiler/commentOnAmbientClass1.ts] ////
 
 //// [a.ts]
-/*! Keep this pinned comment */
+/*!=========
+    Keep this pinned comment
+   =========
+*/
+
+/*! Don't keep this pinned comment */
```

Comment:
Why would we not keep the pinned comment.  The ```/*!``` is explicitly there to say "keep me" :)   It is much simpler if we just keep pinned comments.  Also, it means we can treat any case where we don't keep a pinned comment around as a bug :)